<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Meet Your Faculty | Infectious Disease Genomic Epidemiology 2024</title>
  <meta name="description" content="Workshop website for the 2024 Infectious Disease Genomic Epidemiology Canadian Bioinformatics Workshop" />
  <meta name="generator" content="bookdown 0.43 and GitBook 2.6.7" />

  <meta property="og:title" content="Meet Your Faculty | Infectious Disease Genomic Epidemiology 2024" />
  <meta property="og:type" content="book" />
  <meta property="og:image" content="file:///Users/clin/Documents/CBWGitHub/IDE-2024/docs/index.html/img/bioinformatics_logo.png" />
  <meta property="og:description" content="Workshop website for the 2024 Infectious Disease Genomic Epidemiology Canadian Bioinformatics Workshop" />
  <meta name="github-repo" content="bioinformaticsdotca/IDE_2024#" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Meet Your Faculty | Infectious Disease Genomic Epidemiology 2024" />
  
  <meta name="twitter:description" content="Workshop website for the 2024 Infectious Disease Genomic Epidemiology Canadian Bioinformatics Workshop" />
  <meta name="twitter:image" content="file:///Users/clin/Documents/CBWGitHub/IDE-2024/docs/index.html/img/bioinformatics_logo.png" />

<meta name="author" content="Faculty: William Hsiao, Fiona Brinkman, Emma Griffiths, Jared Simpson, Ed Taboada, Andrew McArthur, Finlay Maguire, Gary Van Domselaar, Robert Beiko" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon" />
<link rel="prev" href="index.html"/>
<link rel="next" href="data-and-compute-setup.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css">



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://bioinformaticsdotca.github.io/">
 <img src="img/bioinformatics_logo.png" width="90%" alt="bioinformatics.ca logo" style="display:block; margin-left:auto; margin-right:auto; margin-top:15px;">
 </a>
<li><center><strong><a href="./"><br>Infectious Disease Genomic Epidemiology 2024<br><br></a></strong></center></li>
<a href="./">
 <img src="img/CBW_epidemiology_icon.png" width="70%" alt="Infectious Disease Genomic Epidemioligy 2024 logo" style="display:block; margin-left:auto; margin-right:auto; ">
 </a>
<br>

<li class="divider"></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Workshop Info</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#class-photo"><i class="fa fa-check"></i>Class Photo</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#schedule"><i class="fa fa-check"></i>Schedule</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#pre-work"><i class="fa fa-check"></i>Pre-work</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="meet-your-faculty.html"><a href="meet-your-faculty.html"><i class="fa fa-check"></i>Meet Your Faculty</a></li>
<li class="chapter" data-level="" data-path="data-and-compute-setup.html"><a href="data-and-compute-setup.html"><i class="fa fa-check"></i>Data and Compute Setup</a></li>
<li class="part"><span><b>II Modules</b></span></li>
<li class="chapter" data-level="" data-path="module-1-introduction-to-genomic-epidemiology.html"><a href="module-1-introduction-to-genomic-epidemiology.html"><i class="fa fa-check"></i>Module 1: Introduction to Genomic Epidemiology</a>
<ul>
<li class="chapter" data-level="" data-path="module-1-introduction-to-genomic-epidemiology.html"><a href="module-1-introduction-to-genomic-epidemiology.html#lecture"><i class="fa fa-check"></i>Lecture</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html"><i class="fa fa-check"></i>Module 2: De Novo Genome Assembly and Annotation</a>
<ul>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#lecture-1"><i class="fa fa-check"></i>Lecture</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#lab"><i class="fa fa-check"></i>Lab</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#integrative-assignment-1"><i class="fa fa-check"></i>Integrative Assignment 1</a>
<ul>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#introduction"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#required-tools"><i class="fa fa-check"></i>Required Tools</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#core-genome-snv-analysis"><i class="fa fa-check"></i>Core Genome SNV Analysis</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#phylogenetic-analysis-visualization"><i class="fa fa-check"></i>Phylogenetic Analysis &amp; Visualization</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#genome-annotation"><i class="fa fa-check"></i>Genome Annotation</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#integrative-assignment-2"><i class="fa fa-check"></i>Integrative Assignment 2</a>
<ul>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#introduction-1"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#list-of-software-utilized"><i class="fa fa-check"></i>List of Software Utilized</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#assignment-setup"><i class="fa fa-check"></i>Assignment Setup</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#phylogenetic-construction"><i class="fa fa-check"></i>Phylogenetic Construction</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#nextclade-analysis"><i class="fa fa-check"></i>Nextclade Analysis</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#recombination-analysis"><i class="fa fa-check"></i>Recombination Analysis</a></li>
<li class="chapter" data-level="" data-path="module-2-de-novo-genome-assembly-and-annotation.html"><a href="module-2-de-novo-genome-assembly-and-annotation.html#answer-key"><i class="fa fa-check"></i>Answer Key</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-3-data-curation-and-data-sharing.html"><a href="module-3-data-curation-and-data-sharing.html"><i class="fa fa-check"></i>Module 3: Data Curation and Data Sharing</a>
<ul>
<li class="chapter" data-level="" data-path="module-3-data-curation-and-data-sharing.html"><a href="module-3-data-curation-and-data-sharing.html#lecture-2"><i class="fa fa-check"></i>Lecture</a></li>
<li class="chapter" data-level="" data-path="module-3-data-curation-and-data-sharing.html"><a href="module-3-data-curation-and-data-sharing.html#lab-1"><i class="fa fa-check"></i>Lab</a>
<ul>
<li class="chapter" data-level="" data-path="module-3-data-curation-and-data-sharing.html"><a href="module-3-data-curation-and-data-sharing.html#data-curation-and-sharing-lab"><i class="fa fa-check"></i>Data Curation and Sharing Lab</a></li>
<li class="chapter" data-level="" data-path="module-3-data-curation-and-data-sharing.html"><a href="module-3-data-curation-and-data-sharing.html#excercise-1-scenario"><i class="fa fa-check"></i>Excercise 1 Scenario</a></li>
<li class="chapter" data-level="" data-path="module-3-data-curation-and-data-sharing.html"><a href="module-3-data-curation-and-data-sharing.html#lab-instructions"><i class="fa fa-check"></i>Lab Instructions</a></li>
<li class="chapter" data-level="" data-path="module-3-data-curation-and-data-sharing.html"><a href="module-3-data-curation-and-data-sharing.html#cancogen-template-curation-reference-guide"><i class="fa fa-check"></i>CanCOGeN Template Curation Reference Guide</a></li>
<li class="chapter" data-level="" data-path="module-3-data-curation-and-data-sharing.html"><a href="module-3-data-curation-and-data-sharing.html#ebi-ols-exercise"><i class="fa fa-check"></i>EBI OLS Exercise</a></li>
<li class="chapter" data-level="" data-path="module-3-data-curation-and-data-sharing.html"><a href="module-3-data-curation-and-data-sharing.html#who-guiding-principles-for-genomic-data-sharing"><i class="fa fa-check"></i>WHO Guiding Principles for Genomic Data Sharing</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-phylogenetic-analysis.html"><a href="module-4-phylogenetic-analysis.html"><i class="fa fa-check"></i>Module 4: Phylogenetic Analysis</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-phylogenetic-analysis.html"><a href="module-4-phylogenetic-analysis.html#lecture-3"><i class="fa fa-check"></i>Lecture</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html"><a href="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html"><i class="fa fa-check"></i>Module 5: Viral Pathogen Genomic Analysis - Single Nucleotide Variants</a>
<ul>
<li class="chapter" data-level="" data-path="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html"><a href="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html#lecture-4"><i class="fa fa-check"></i>Lecture</a></li>
<li class="chapter" data-level="" data-path="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html"><a href="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html#lab-2"><i class="fa fa-check"></i>Lab</a>
<ul>
<li class="chapter" data-level="" data-path="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html"><a href="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html#introduction-2"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html"><a href="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html#data-sets"><i class="fa fa-check"></i>Data Sets</a></li>
<li class="chapter" data-level="" data-path="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html"><a href="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html#data-download-and-preparation"><i class="fa fa-check"></i>Data Download and Preparation</a></li>
<li class="chapter" data-level="" data-path="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html"><a href="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html#running-signal"><i class="fa fa-check"></i>Running SIGNAL</a></li>
<li class="chapter" data-level="" data-path="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html"><a href="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html#reference-based-assembly-using-signal"><i class="fa fa-check"></i>Reference-based Assembly using SIGNAL</a></li>
<li class="chapter" data-level="" data-path="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html"><a href="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html#quality-control-and-assessment"><i class="fa fa-check"></i>Quality Control and Assessment</a></li>
<li class="chapter" data-level="" data-path="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html"><a href="module-5-viral-pathogen-genomic-analysis---single-nucleotide-variants.html#coverage-analysis"><i class="fa fa-check"></i>Coverage Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html"><a href="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html"><i class="fa fa-check"></i>Module 6: Bacterial Pathogen Genomic Analysis - Genome-based Subtyping</a>
<ul>
<li class="chapter" data-level="" data-path="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html"><a href="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html#lecture-5"><i class="fa fa-check"></i>Lecture</a></li>
<li class="chapter" data-level="" data-path="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html"><a href="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html#lab-3"><i class="fa fa-check"></i>Lab</a>
<ul>
<li class="chapter" data-level="" data-path="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html"><a href="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html#introduction-3"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html"><a href="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html#learning-objectives"><i class="fa fa-check"></i>Learning Objectives</a></li>
<li class="chapter" data-level="" data-path="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html"><a href="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html#timeline-for-training"><i class="fa fa-check"></i>Timeline for Training</a></li>
<li class="chapter" data-level="" data-path="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html"><a href="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html#getting-started"><i class="fa fa-check"></i>Getting Started</a></li>
<li class="chapter" data-level="" data-path="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html"><a href="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html#day-1-pre-processing-and-preliminary-analyses-1"><i class="fa fa-check"></i>Day 1: Pre-processing and Preliminary Analyses</a></li>
<li class="chapter" data-level="" data-path="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html"><a href="module-6-bacterial-pathogen-genomic-analysis---genome-based-subtyping.html#day-2-visualizations-to-facilitate-analysis-1"><i class="fa fa-check"></i>Day 2: Visualizations to Facilitate Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-7-antimicrobila-resistant-gene-amr-analysis.html"><a href="module-7-antimicrobila-resistant-gene-amr-analysis.html"><i class="fa fa-check"></i>Module 7: Antimicrobila Resistant Gene (AMR) Analysis</a>
<ul>
<li class="chapter" data-level="" data-path="module-7-antimicrobila-resistant-gene-amr-analysis.html"><a href="module-7-antimicrobila-resistant-gene-amr-analysis.html#lecture-6"><i class="fa fa-check"></i>Lecture</a></li>
<li class="chapter" data-level="" data-path="module-7-antimicrobila-resistant-gene-amr-analysis.html"><a href="module-7-antimicrobila-resistant-gene-amr-analysis.html#lab-4"><i class="fa fa-check"></i>Lab</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-8-phylodynamics.html"><a href="module-8-phylodynamics.html"><i class="fa fa-check"></i>Module 8: Phylodynamics</a>
<ul>
<li class="chapter" data-level="" data-path="module-8-phylodynamics.html"><a href="module-8-phylodynamics.html#lecture-7"><i class="fa fa-check"></i>Lecture</a></li>
<li class="chapter" data-level="" data-path="module-8-phylodynamics.html"><a href="module-8-phylodynamics.html#lab-5"><i class="fa fa-check"></i>Lab</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-9-mobile-genetic-elements.html"><a href="module-9-mobile-genetic-elements.html"><i class="fa fa-check"></i>Module 9: Mobile Genetic Elements</a>
<ul>
<li class="chapter" data-level="" data-path="module-9-mobile-genetic-elements.html"><a href="module-9-mobile-genetic-elements.html#lecture-8"><i class="fa fa-check"></i>Lecture</a></li>
<li class="chapter" data-level="" data-path="module-9-mobile-genetic-elements.html"><a href="module-9-mobile-genetic-elements.html#lab-6"><i class="fa fa-check"></i>Lab</a>
<ul>
<li class="chapter" data-level="" data-path="module-9-mobile-genetic-elements.html"><a href="module-9-mobile-genetic-elements.html#data-files"><i class="fa fa-check"></i>Data Files</a></li>
</ul></li>
</ul></li>
<li class="divider"></li>
<h1 id="sponsors">Sponsors</h1>
<center>
<a href="https://oicr.on.ca/"><img src="img/sponsors/oicr.png" width=25% style="display:inline-block;" style="padding-bottom: 20px" alt="Ontario Institute for Cancer Research logo.">
<a href="https://www.hpcforhealth.ca/"><img src="img/sponsors/hpc4health.png" width=60% style="display:inline-block;" style="padding-bottom: 20px "alt="HPC4Health logo.">
<br><br>
<a href="https://genomecanada.ca/"><img src="img/sponsors/genomecanada.png" height="70" style="display:inline-block;"  style="padding-bottom: 20px" alt="Genome Canada logo.">
<a href="https://www.ontariogenomics.ca/"><img src="img/sponsors/ontariogenomics.png" height="70" style="display:inline-block;" style="padding-bottom: 20px;" alt="Ontario Genomics logo.">
<br><br>
<a href="https://www.cihr-irsc.gc.ca"><img src="img/sponsors/c3g.png" width=50% style="padding-bottom: 20px" alt="Canadian Centre for Computational Genomics logo."></a>
<a href="https://computationalgenomics.ca/"><img src="img/sponsors/cihr.png" width=50% style="padding-bottom: 20px" alt="Canadian Institutes of Health Research logo."></a>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Infectious Disease Genomic Epidemiology 2024</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="meet-your-faculty" class="section level1 hasAnchor">
<h1>Meet Your Faculty<a href="meet-your-faculty.html#meet-your-faculty" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="william-hsiao" class="section level4 hasAnchor">
<h4>William Hsiao<a href="meet-your-faculty.html#william-hsiao" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Associate Professor <br>
Faculty of Health Sciences, Simon Fraser University <br>
Vancouver, BC, CA</p>
<p><a href="mailto:wwhsiao@sfu.ca" class="email">wwhsiao@sfu.ca</a> <br>
Website: <a href="https://cidgoh.ca/" class="uri">https://cidgoh.ca/</a></p>
</blockquote>
<p>William Hsiao is a public health infectious disease researcher with a background in microbial
genomics and bioinformatics. He is an associate professor in the Faculty of Health Sciences
at Simon Fraser University and an affiliated researcher at BCCDC Public Health Laboratory
and at Canada’s Michael Smith Genome Sciences Centre. Will leads an interdisciplinary
group of researchers interested in solving practical public health and animal health problems
through a One Health lens at the Centre for Infectious Disease Genomics and One Health.</p>
</div>
<div id="jared-simpson" class="section level4 hasAnchor">
<h4>Jared Simpson<a href="meet-your-faculty.html#jared-simpson" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Principal Investigator, Ontario Institute for Cancer Research <br>
Assistant Prof. Department of Computer Science, University of Toronto <br></p>
<p>Website: <a href="https://simpsonlab.github.io/" class="uri">https://simpsonlab.github.io/</a></p>
</blockquote>
<p>Dr. Simpson develops algorithms and software for the analysis of high-throughput sequencing
data. He is interested in de novo assembly and the detection of sequence variation in
individuals, cancers and populations, with a focus on long read sequencing technologies. Dr.
Simpson developed the ABYSS, SGA and nanopolish software packages.</p>
</div>
<div id="fiona-brinkman" class="section level4 hasAnchor">
<h4>Fiona Brinkman<a href="meet-your-faculty.html#fiona-brinkman" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Distinguished Professor, FRSC, Department of Molecular Biology and Biochemistry <br>
Associate Member, School of Computing Science and Faculty of Health Sciences,
Simon Fraser Univeristy <br>
Burnaby, BC, Canada</p>
<p><a href="mailto:brinkman@sfu.ca" class="email">brinkman@sfu.ca</a> <br>
Website: <a href="https://www.brinkmanlab.ca/" class="uri">https://www.brinkmanlab.ca/</a></p>
</blockquote>
<p>Dr. Brinkman is developing bioinformatic resources to better track infectious diseases using
genomic data, and improve prediction of new vaccine/drug targets. Her primary aim is to
develop more sustainable, integrated approaches for infectious disease control, however she
is also applying her methods to aid allergy, child health, and environmental research.</p>
</div>
<div id="rob-beiko" class="section level4 hasAnchor">
<h4>Rob Beiko<a href="meet-your-faculty.html#rob-beiko" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Professor <br>
Faculty of Computer Science, Dalhousie University <br>
Halifax, NS, Canada</p>
<p><a href="mailto:rbeiko@dal.ca" class="email">rbeiko@dal.ca</a></p>
</blockquote>
<p>Rob Beiko is a professor in bioinformatics whose research encompasses microbial evolution
and gene transfer, comparative genomics, and microbial community analysis. He is the lead
of the ARETE project, which is developing a software pipeline for annotation of AMR and
mobile genetic elements, phylogenomic analysis, and inference of recombination and transfer
in large sets of pathogen genomes. He has also developed and contributed to software tools
including STAMP for statistical analysis of metagenome profiles, rSPR for efficient inference
of gene transmission, and PICRUSt for the prediction of metagenome composition based on
marker-gene surveys.</p>
</div>
<div id="finlay-maguire" class="section level4 hasAnchor">
<h4>Finlay Maguire<a href="meet-your-faculty.html#finlay-maguire" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Assistant Professor <br>
Faculty of Computer Science and Department of Community Health &amp; Epidemiology, Dalhousie Univerisity <br>
Halifax, NS, Canada</p>
<p><a href="mailto:finlay.maguire@dal.ca" class="email">finlay.maguire@dal.ca</a> <br>
Website: finlaymagui.re</p>
</blockquote>
<p>Finlay Maguire is a genomic epidemiologist whose work centers on leveraging data in
innovative ways to answer questions related to applied health and social issues. This includes
developing bioinformatics methods to more effectively use genomic data to mitigate infectious
diseases and broad interdisciplinary collaborations in areas such as refugee healthcare
provision and online radicalisation. They are an active contributor to the national and
international public health responses to emerging viral zoonoses and antimicrobial resistance,
co-chair of the PHA4GE data structures working group, and act as a Pathogenomics
Bioinformatics Lead for Sunnybrook’s Shared Hospital Laboratory.</p>
</div>
<div id="gary-van-domselaar" class="section level4 hasAnchor">
<h4>Gary Van Domselaar<a href="meet-your-faculty.html#gary-van-domselaar" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Chief, Bioinformatics <br>
National Microbiology Laboratory <br>
Public Health Agency of Canada <br>
Winnipeg, MB, Canada</p>
<p><a href="mailto:gary.vandomselaar@phac-aspc.gc.ca" class="email">gary.vandomselaar@phac-aspc.gc.ca</a></p>
</blockquote>
<p>Dr. Gary Van Domselaar, PhD (University of Alberta, 2003) is the Chief of the Bioinformatics
Section at the National Microbiology Laboratory in Winnipeg Canada and Associate Professor in
the Department of Medical Microbiology and Infectious Diseases at the University of Manitoba. Dr.
Van Domselaar’s lab develops bioinformatics methods and pipelines to understand, track, and
control circulating infectious diseases in Canada and globally. His research and development
activities span metagenomics, infectious disease genomic epidemiology, genome annotation,
population structure analysis, and microbial genome wide association studies.</p>
</div>
<div id="ed-taboada" class="section level4 hasAnchor">
<h4>Ed Taboada<a href="meet-your-faculty.html#ed-taboada" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Research Scientist <br>
Genomic Epidemiology Research Unit <br>
Division of Enteric Diseases <br>
National Microbiology Laboratory <br>
Public Health Agency of Canada <br>
Winnipeg, MB, Canada</p>
<p><a href="mailto:eduardo.taboada@phac-aspc.gc.ca" class="email">eduardo.taboada@phac-aspc.gc.ca</a></p>
</blockquote>
<p>Eduardo Taboada is a research scientist in the Division of Enteric Diseases at the NML. Since
obtaining his PhD in molecular genetics at the Department of Biology, University of Ottawa in
1999, he has applied his expertise in molecular biology and population genetics towards the
development of comparative genomics and bioinformatics-based tools for studying various
aspects of the biology, ecology, and epidemiology of bacterial food- and waterborne
pathogens, with a focus on Campylobacter, Salmonella, and E. coli. Over these past two
decades, Ed has led or participated in several projects funded by Genome Canada and the
federal government’s Genomics Research and Development Initiative. His current research
focus is on developing quantitative tools for integrated genomic and epidemiologic analysis
and the application of analytical epidemiology approaches to genomic datasets.</p>
</div>
<div id="emma-griffiths" class="section level4 hasAnchor">
<h4>Emma Griffiths<a href="meet-your-faculty.html#emma-griffiths" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Research Associate <br>
Faculty of Health Sciences, Simon Fraser University <br>
Vancouver, BC, Canada</p>
<p><a href="mailto:emma_griffiths@sfu.ca" class="email">emma_griffiths@sfu.ca</a></p>
</blockquote>
<p>Emma Griffiths is a research associate at the Centre for Infectious Disease Genomics and
One Health (CIDGOH) in the Faculty of Health Sciences at Simon Fraser University in
Vancouver, Canada. Her work focuses on developing and implementing ontologies and data
standards for public health and food safety genomics to help improve data harmonization and
integration. She is a member of the Standards Council of Canada and leads the Public Health
Alliance for Genomic Epidemiology (PHA4GE) Data Structures Working Group.</p>
</div>
<div id="andrew-mcarthur" class="section level4 hasAnchor">
<h4>Andrew McArthur<a href="meet-your-faculty.html#andrew-mcarthur" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Professor <br>
Michael G. DeGroote Institute for Infectious Disease Research <br>
Department of Biochemistry and Biomedical Sciences <br>
McMaster University <br>
Hamilton, Ontario, Canada</p>
</blockquote>
<p>Dr. McArthur is a Professor and David Braley Chair in Computational Biology at McMaster
University. Dr. McArthur has had a career in the United States and Canada, including
NIH-funded positions at the Marine Biological Laboratory and Brown University, where he led
the genome assembly of the diarrheal pathogen Giardia intestinalis, plus 10 years of
experience in the private sector. Dr. McArthur’s research team focuses on building tools,
databases, and algorithms for the genomic surveillance of infectious pathogens. He and his
team developed the Comprehensive Antibiotic Resistance Database (card.mcmaster.ca) and
the SARS-CoV-2 Illumina GeNome Assembly Line software platform.</p>
</div>
<div id="darian-hole" class="section level4 hasAnchor">
<h4>Darian Hole<a href="meet-your-faculty.html#darian-hole" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Bioinformatician <br>
Computational and Operational Genomics Unit <br>
National Microbiology Laboratory <br>
Public Health Agency of Canada <br>
Winnipeg, MB, Canada</p>
<p><a href="mailto:darian.hole@phac-aspc.gc.ca" class="email">darian.hole@phac-aspc.gc.ca</a></p>
</blockquote>
<p>Darian Hole is a bioinformatician at the Public Health Agency of Canada working under Dr.
Andrea Tyler and Dr. Ana Duggan. He focuses on genomic analysis process development,
validation, and automation along with assisting partner labs in bringing in genomics to their
workflows. Darian has mostly focused on viral diseases the past 5 years working at
implementing standards, automation, and guidelines for SARS-CoV-2, Mpox, Polio, and
Influenza.</p>
</div>
<div id="karyn-mukiri" class="section level4 hasAnchor">
<h4>Karyn Mukiri<a href="meet-your-faculty.html#karyn-mukiri" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>PhD Graduate Student Researcher <br>
Faculty of Health Sciences, McMaster University <br>
Department of Biochemistry and Biomedical Sciences <br>
Hamilton, ON, Canada</p>
<p><a href="mailto:mukirikm@mcmaster.ca" class="email">mukirikm@mcmaster.ca</a></p>
</blockquote>
<p>Karyn is a PhD student under the supervision of Dr. Andrew McArthur where her main
research focus is on the development of predictive genomic annotation algorithms to increase
antibiotic resistance surveillance by the Resistance Gene Identifier. She is also a McMaster
alum, having completed her undergraduate degree in Biotechnology under the Faculty of
Engineering. Although the bulk of her work focuses on more algorithmic problems, her current
research efforts are in helping pinpoint instances of miscuration within the Comprehensive
Antibiotic Resistance Database.</p>
</div>
<div id="miguel-prieto" class="section level4 hasAnchor">
<h4>Miguel Prieto<a href="meet-your-faculty.html#miguel-prieto" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>PhD Graduate Student Researcher <br>
Faculty of Health Sciences, Simon Fraser University <br>
Vancouver, BC, Canada</p>
<p><a href="mailto:mprietog@sfu.ca" class="email">mprietog@sfu.ca</a></p>
</blockquote>
<p>Miguel is a first year PhD student studying the impact of human infectious diseases and the
role of the microbiome in humans. Particularly, he aims to use machine learning and
metagenomic analysis to predict poor outcomes in chronic respiratory diseases. Miguel is a
Medical Doctor from the Universidad del Valle in Colombia and has worked as a researcher
and project coordinator on translational studies of neglected tropical diseases like
leishmaniasis. Before coming to SFU, he completed a Master of Experimental Medicine at
UBC exploring blood-based gene expression biomarkers of nontuberculous mycobacterial
infections in patients with cystic fibrosis.</p>
</div>
<div id="madeline-mccarthy" class="section level4 hasAnchor">
<h4>Madeline McCarthy<a href="meet-your-faculty.html#madeline-mccarthy" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>PhD Graduate Student Researcher <br>
Department of Biochemistry and Biomedical Sciences <br>
McMaster University <br>
Hamilton, ON, Canada</p>
<p><a href="mailto:mccarm15@mcmaster.ca" class="email">mccarm15@mcmaster.ca</a></p>
</blockquote>
<p>Madeline is a PhD Candidate in the McArthur Lab at McMaster University. Her work focuses
on the development of targeted, cost-effective methods for culture-free outbreak detection and
surveillance of bacterial pathogens. In addition to whole genome capture, she is testing the
feasibility of targeted, culture-free plasmid recovery using bait capture and long-read
sequencing. Prior to her PhD, she completed a MSc in microbiology at the University of
Saskatchewan where she studied E. coli outbreaks in broiler chickens before hanging up her
pipettes and switching to bioinformatics.</p>
</div>
<div id="charlie-barclay" class="section level4 hasAnchor">
<h4>Charlie Barclay<a href="meet-your-faculty.html#charlie-barclay" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>MSc Graduate Student Researcher <br>
Department of Physics and Astronomy <br>
University of British Columbia <br>
Hamilton, ON, Canada</p>
<p><a href="mailto:cbarcl01@mail.ubc.ca" class="email">cbarcl01@mail.ubc.ca</a></p>
</blockquote>
<p>Charlie is an ontology curator at the Centre for Infectious Disease Genomics and One Health
(CIDGOH), focusing on data standards for contextual data of genomic epidemiology, including
wastewater surveillance. With five years of experience in data management, specializing in
biodiversity and genomics data, she also actively contributes to the Public Health Alliance for
Genomic Epidemiology (PHA4GE) and the Global Alliance for Genomics and Health
(GA4GH).</p>
</div>
<div id="michelle-brazas" class="section level4 hasAnchor">
<h4>Michelle Brazas<a href="meet-your-faculty.html#michelle-brazas" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Scientific Director <br>
Canadian Bioinformatics Workshops (CBW) <br>
Toronto, ON, CA</p>
<p><a href="mailto:support@bioinformatics.ca" class="email">support@bioinformatics.ca</a></p>
</blockquote>
<p>Dr. Michelle Brazas is the Associate Director for Adaptive Oncology at the Ontario Institute for
Cancer Research (OICR), and acting Scientific Director at Bioinformatics.ca. Previously, Dr.
Brazas was the Program Manager for Bioinformatics.ca and a faculty member in
Biotechnology at BCIT. Michelle co-founded and runs the Toronto Bioinformatics User Group
(TorBUG) now in its 11th season, and plays an active role in the International Society of
Computational Biology where she sits on the Board of Directors and Executive Board.</p>
</div>
<div id="nia-hughes" class="section level4 hasAnchor">
<h4>Nia Hughes<a href="meet-your-faculty.html#nia-hughes" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Program Manager, Bioinformatics.ca <br>
Ontario Institute for Cancer Research <br>
Toronto, ON, Canada <br></p>
<p><a href="mailto:nia.hughes@oicr.on.ca" class="email">nia.hughes@oicr.on.ca</a></p>
</blockquote>
<p>Nia is the Program Manager for Bioinformatics.ca, where she coordinates the Canadian
Bioinformatics Workshop Series. Prior to starting at OICR, she completed her M.Sc. in
Bioinformatics from the University of Guelph in 2020 before working there as a
bioinformatician studying epigenetic and transcriptomic patterns across maize varieties.</p>
</div>
<div id="zhibin-lu" class="section level4 hasAnchor">
<h4>Zhibin Lu<a href="meet-your-faculty.html#zhibin-lu" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/MISSINGHEADSHOT.png" width="200" alt="Photo of NAME"></p>
<blockquote>
<p>Senior Manager, Digital Research <br>
University Health Network <br>
Toronto, ON, Canada <br></p>
<p><a href="mailto:zhibin@gmail.com" class="email">zhibin@gmail.com</a></p>
</blockquote>
<p>Zhibin Lu is a senior manager at University Health Network Digital. He is responsible for UHN
HPC operations and scientific software. He manages two HPC clusters at UHN, including
system administration, user management, and maintenance of bioinformatics tools for
HPC4health. He is also skilled in Next-Gen sequence data analysis and has developed and
maintained bioinformatics pipelines at the Bioinformatics and HPC Core. He is a member of
the Digital Research Alliance of Canada Bioinformatics National Team and Scheduling
National Team.</p>

<!--
IF YOUR WORKSHOP DOES NOT INCLUDE COMPUTE:
- REMOVE "AND COMPUTE SETUP" FROM THE TOP-LEVEL HEADER
- DELETE THE LAST SECTION (COMPUTE SETUP)

IF YOUR WORKSHOP INCLUDES COMPUTE:
- AFTER THE WORKSHOP, ADD DATA DOWNLOAD LINKS, AWS MACHINE IMAGE INSTRUCTIONS, ETC.
- IF IT IS MORE APPROPRIATE, YOU CAN INCLUDE THE DATA DOWNLOAD LINKS IN EACH OF THE MODULES.
-->
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="data-and-compute-setup.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": "https://github.com/cbw-dev/IDE-2024/blob/main/001-faculty.Rmd",
    "text": "Edit"
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": ["_main.pdf", "_main.epub"],
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

</body>

</html>
